design-system/materials/internals/definition.yaml

```diff
@@ -492,6 +500,8 @@ decisionGroupsByTheme:
           choice: shadow-9
         shadow-for-button-when-disabled:
           choice: '0 0 0 1px var(--color-neutral) inset'
+        shadow-for-card-when-raised:
+          choice: '0 1px 2px rgb(25 40 81 / 5%)'
```

Comment:
This token is inside the `default` theme group so the value here should be `shadow-1` (the current value).
Also, you would need to add this token in the test group with this value (`0 1px 2px rgb(25 40 81 / 5%)`).